File: ios/Classes/data/PluginFolder.swift
//
//  PluginDataSet.swift
//  zpdl_studio_media_plugin
//
//  Created by 김경환 on 2020/10/12.
//

struct PluginFolder: PluginToMap {
    let id: String
    let displayName: String
    let count: Int

    func pluginToMap() -> [String : Any] {
        return [
            "id": id,
            "displayName": displayName,
            "count": count
        ]
    }
}

File: android/src/main/kotlin/com/zpdl_studio/zpdl_studio_media_plugin/data/PluginFolder.kt
package com.zpdl_studio.zpdl_studio_media_plugin.data

data class PluginFolder(
    val id: String,
    val displayName: String,
    val count: Int,
    val modifyTimeMs: Long
) : PluginToMap {
    override fun pluginToMap(): Map<String, *> = hashMapOf(
            "id" to id,
            "displayName" to displayName,
            "count" to count,
            "modifyTimeMs" to modifyTimeMs,
    )
}